bin/templatemap_generator.php

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
```

Comment:
Can we please leave these things out of the skeleton? Seems excessive to
me, at least...
On Jun 12, 2016 2:17 PM, "Abdul Malik Ikhsan" <notifications@github.com>
wrote:

> In bin/templatemap_generator.php
> <https://github.com/zendframework/ZendSkeletonApplication/pull/340#discussion_r66720133>
> :
>
> > @@ -0,0 +1,256 @@
> > +#!/usr/bin/env php
> > +<?php
> > +/**
> > + * Zend Framework (http://framework.zend.com/)
> > + *
> > + * @link      http://github.com/zendframework/zf2 for the canonical source repository
>
> https://github.com/zendframework/ZendSkeletonApplication
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendframework/ZendSkeletonApplication/pull/340/files/96faacadc30c73e013b3b5470de7acf968652da8#r66720133>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJakBO_Kc3-gGKPRsl6Csr-fIZqyJGxks5qK_jhgaJpZM4Izev->
> .
>